### PR TITLE
[FLINK-19431][docs]Translate "Monitoring REST API" page of "Debugging & Monitoring" into Chinese

### PR DESCRIPTION
**Pull request by weizheng92**
## What is the purpose of the change

Translate "Monitoring REST API" page of "Debugging & Monitoring" into Chinese

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by weizheng92** [comment]
@klion26  The project was re-forked for some reason, could you please review it again? thanks

File: docs/monitoring/rest_api.zh.md [hunk 1]
```diff
@@ -22,46 +22,45 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Flink has a monitoring API that can be used to query status and statistics of running jobs, as well as recent completed jobs.
-This monitoring API is used by Flink's own dashboard, but is designed to be used also by custom monitoring tools.
+Flink 具有监控 API ，可用于查询正在运行的作业以及最近完成的作业的状态和统计信息。该监控 API 被用于 Flink 自己的仪表盘，同时也可用于自定义监控工具。
 
-The monitoring API is a REST-ful API that accepts HTTP requests and responds with JSON data.
+该监控 API 是 REST-ful 风格的 API ，可以接受 HTTP 请求并返回 JSON 格式的数据。
```
Comment (klion26) [inline docs/monitoring/rest_api.zh.md:27]: `该监控 API 是 REST-ful 风格的 API` -> `该监控 API 是 REST-ful 风格的` 是否就可以了

File: docs/monitoring/rest_api.zh.md [hunk 2]
```diff
@@ -22,46 +22,45 @@ specific language governing permissions and limitations
...
 
 * This will be replaced by the TOC
 {:toc}
 
 
-## Overview
+## 概览
```
Comment (klion26) [inline docs/monitoring/rest_api.zh.md:33]: 请在每个标题前面添加下 `<a>` 标签吧，具体参考 [wiki](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications)
链接的锚点可以参考英文版的 url

File: docs/monitoring/rest_api.zh.md [hunk 3]
```diff
@@ -22,46 +22,45 @@ specific language governing permissions and limitations
...
-We use *Netty* and the *Netty Router* library to handle REST requests and translate URLs. This choice was made because this combination has lightweight dependencies, and the performance of Netty HTTP is very good.
+我们使用 *Netty* 和 *Netty Router* 库来处理 REST 请求和转换 URL 。选择该选项是因为这种组合具有轻量级依赖关系，并且 Netty HTTP 的性能非常好。
 
-To add new requests, one needs to
-* add a new `MessageHeaders` class which serves as an interface for the new request,
-* add a new `AbstractRestHandler` class which handles the request according to the added `MessageHeaders` class,
-* add the handler to `org.apache.flink.runtime.webmonitor.WebMonitorEndpoint#initializeHandlers()`.
+添加新的请求，需要
+* 添加一个新的 `MessageHeaders` 类，作为新请求的接口，
+* 添加一个新的 `AbstractRestHandler` 类，该类根据添加的 `MessageHeaders` 类处理请求，
```
Comment (klion26) [inline docs/monitoring/rest_api.zh.md:48]: “该类根据添加的 `MessageHeaders` 类处理请求” 这句读起来有点怪怪的，能否再优化下

**Review by klion26 — COMMENTED** [review]
@weizheng92 thanks for your contribution, left some comments, please take a look.

**Comment by weizheng92** [comment]
@klion26 感谢您的回复，已做修改，麻烦您有时间再看一下，谢谢。

**Review by klion26 — APPROVED** [review]
@weizheng92 thanks for the update, LGTM now
cc @wuchong @rmetzger

**Comment by weizheng92** [comment]
@wuchong @rmetzger  klion26已经review通过了，有时间看下吗

**Pull request merged**